frontend/constants.js

```diff
@@ -0,0 +1,4 @@
+import getConfig from './helpers/getConfig';
+
+export const { searchBarBlacklist } = getConfig();
```

Comment:
since this is a constant I think it should have a name like SEARCH_BAR_BLACKLIST. You should still be able to use the object deconstruction by doing something like 
````
export const { searchBarBlacklist: SEARCH_BAR_BLACKLIST } = getConfig();
````
And if you are paranoid like me you can even assign a default value here like 
````
export const { searchBarBlacklist: SEARCH_BAR_BLACKLIST = [] } = getConfig();
````

Reply:
Sounds good 👍